Add extension name to notice/remove config array (#24)
WPWhiteSecurity/wsal-extension-template: README.md

    */

    /*

>      REQUIRED. Here we include and fire up the main core class. This will crucial so leave in tact.

Reviewer: ```suggestion
     REQUIRED. Here we include and fire up the main core class. This is crucial so leave intact.
```
